.github/workflows/ai-revision.yaml

```diff
@@ -0,0 +1,61 @@
+name: ai-revision
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to revise'
+        required: true
+        type: string
+        default: 'main'
+      file_names:
+        description: 'File names to revise'
+        required: false
+        type: string
+        default: ''
+      model:
+        description: 'Language model'
+        required: true
+        type: string
+        default: 'text-davinci-003'
+      branch_name:
+        description: 'Output branch'
+        required: true
+        type: string
+        default: 'ai-revision-davinci'
+
+jobs:
+  ai-revise:
+    name: AI Revise
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash --login {0}
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with: 
+          ref: ${{ inputs.branch }} 
+      - name: Install environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Install Manubot AI revision dependencies
+        run: pip install git+https://github.com/manubot/manubot@d4242ffa4194e4a13a68c5f6466feff559d3f9d5#egg=manubot[ai-rev]
```

Comment:
`pip install manubot[ai-rev]` cannot be used here because the latest version does not include the `ai-rev` part.

These lines work but have an important issue. As with the previous step, we should read the changeset hash from `build/environment.yml`, since otherwise, we should always update here when the manubot version in the environment is changed. The other solution is to install the latest version of manubot, but that's not how it is installed in `build/environment.yml`.

Reply:
We can and should make a manubot release. I'm okay with this as is.

Reply:
Now that we have a new Manubot release with this feature, should we consider installing the latest Manubot from PyPI for this workflow? I agree with @miltondp that it would be inelegant to have to update the hash manually in two places every time we update the manubot repo. This workflow could grab the hash from the `build/environment.yml` file, which hurts readability but helps maintainability.

Reply:
I made a change to use the same URL in `build/environment.yml`.